cylc/flow/scripts/validate_reinstall.py

```diff
@@ -166,10 +166,14 @@ async def vr_cli(parser: COP, options: 'Values', workflow_id: str):
         return 1
 
     # Force on the against_source option:
-    options.against_source = True   # Make validate check against source.
+    options.against_source = True
```

Comment:
Would it be better to do

```
original_option_against_source: bool = options.against_source
options.against_source = True

...

# after validation restore the original option value
options.against_source = original_option_against_source
```

Reply:
No - I've just had this discussion with @MetRonnie  - it doesn't make sense to have this argument available in Cylc VR.